data/logement/construction.publicodes

```diff
@@ -1,61 +1,166 @@
   unité: kgCO2e
 
-logement . construction . annuelle et par surface:
-  formule: par surface / durée d'amortissement
+logement . construction . empreinte amortie:
+  formule:
+    somme:
+      - base
+      - rafraichissement
+      - rénovation
+      - déforestation
   unité: kgCO2e/m2
 
-logement . construction . par surface:
+logement . construction . base:
   formule:
     variations:
-      - si:
-          toutes ces conditions:
-            - est défini: logement . appartement
-            - logement . appartement
-        alors: par surface . appartement + déforestation construction
-      - sinon: par surface . maison + déforestation construction
-  unité: kgCO2e/m2
-  note: |
-    Nous avons décidé de supprimer la question liée à l'éco-constructin du logement, car l'empreinte associée faisait débat. Dès que nous retrouverons des données fiables, nous la remettrons en ligne !
-
-# logement . éco-construit:
-#   question: Votre logement est-il fait en bois, paille, pierre ou terre (éco-construit) ?
-#   par défaut: non
-
-logement . construction . déforestation construction: 
-  formule: 44.8
+      - si: logement . type = 'appartement'
+        alors: base . appartement / durée d'amortissement
+      - si: logement . type = 'maison'
+        alors: base . maison / durée d'amortissement
+      - sinon:
+          moyenne:
+            - base . maison / durée d'amortissement
+            - base . appartement / durée d'amortissement
   unité: kgCO2e/m2
-  note: |
-    L'étude Pendril & al. sur la déforestation importée compte une déforestation moyenne de 37 kgCO2e par habitant. 
-    Sur la base d'un logement moyen de 41,3 m² par habitant (statistiques INSEE - 90,9 m² par logement en moyenne pour un foyer de 2,2 personnes), et d'un amortissement de 50 ans, cela donne une empreinte de 44,8 kgCO2e / m².
 
-logement . construction . par surface . appartement:
+logement . construction . base . appartement:
   titre: Empreinte appartement par m2
-  formule: 525
+  formule: 740
   unité: kgCO2e/m2
-  note: Facteur d'émission issu de la Base Empreinte.
 
-logement . construction . par surface . maison:
+logement . construction . base . maison:
   titre: Empreinte maison par m2
-  formule: 425
+  formule: 640
   unité: kgCO2e/m2
-  note: Facteur d'émission issu de la Base Empreinte.
 
-logement . construction . durée d'amortissement:
+logement . construction . base . durée d'amortissement:
   formule: 50
   note: |
     La durée de vie d'un bâtiment est fixée à 50 ans (voir la [documentation de la base empreinte, onglet Achats de bien/ Bâtiments](https://prod-basecarbonesolo.ademe-dri.fr/documentation/UPLOAD_DOC_FR/)).
+
+logement . construction . rafraichissement:
+  titre: Travaux de rafraîchissement
+  icônes: 🎨
+  applicable si: logement . âge >= seuil
+  formule: 40 kgCO2e/m2 / amortissement
```

Comment:
On doit mettre l'unité dans la formule ici ?

Reply:
Oui car si on fait un calcul ici avec "40" sans unité, il signale une erreur d'homogénéité

Reply:
Ah OK. Chelou.